src/NodeDev.Blazor/Services/ServicesExtension.cs

```diff
@@ -8,11 +8,13 @@ public static class ServicesExtension
 
 		public static IServiceCollection AddNodeDev(this IServiceCollection services)
 		{
-			services
+            services
 				.AddMudServices()
-				.AddScoped<DebuggedPathService>();
+				.AddScoped<DebuggedPathService>()
+				.AddSingleton<ProjectService>()
+				.AddSingleton(new AppOptionsContainer ("AppOptions.json"));
```

Comment:
Since the AppOptionsContainer is a service, shouldn't we put it in the Services folder?